From ebc2e26d395ecda2325f44f9c2745102b0c7e111 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jacobo=20Aragunde=20P=C3=A9rez?= <jaragunde@igalia.com>
Date: Wed, 13 Feb 2019 10:54:45 +0100
Subject: [PATCH] Horizontal homescreen

---
 src/db/areas.db | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/src/db/areas.db b/src/db/areas.db
index 03ddfe4..d7aaa53 100644
--- a/src/db/areas.db
+++ b/src/db/areas.db
@@ -5,17 +5,17 @@
             "rect": {
                 "x": 0,
                 "y": 0,
-                "w": 1080,
-                "h": 1920
+                "w": 1920,
+                "h": 1080
             }
         },
         {
             "name": "normal.full",
             "rect": {
-                "x": 0,
-                "y": 218,
-                "w": 1080,
-                "h": 1488
+                "x": 180,
+                "y": 0,
+                "w": 1526,
+                "h": 1080
             }
         },
         {
@@ -75,10 +75,10 @@
         {
             "name": "on_screen",
             "rect": {
-                "x": 0,
-                "y": 218,
-                "w": 1080,
-                "h": 1488
+                "x": 180,
+                "y": 0,
+                "w": 1526,
+                "h": 1080
             }
         }
     ]
-- 
2.11.0

